packages/web/components/layouts/main.tsx

```diff
@@ -182,3 +175,42 @@ const OsmosisFullLogo: FunctionComponent<{
     }}
   />
 );
+
+const Drawer = ({
+  children,
+  showSidebar,
+  isMobile,
+}: {
+  children: ReactNode;
+  showSidebar: boolean;
+  isMobile: boolean;
+}) => {
+  if (!isMobile)
```

Comment:
Nitpicky, but I think it's easier to read and scan to return the true case rather than the opposite case with the `!` operator.